test/test_import_time.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
+
+from torch.testing._internal.common_utils import TestCase, run_tests
+
+
+class TestImportTime(TestCase):
+    # this test could eventually be changed to fail if the import time
+    # is greater than a certain threshold, but for now we just use it as
+    # a way to track the duration of `import torch` in our ossci-metrics
+    # S3 bucket (see tools/print_test_stats.py)
+    def test_time_import_torch(self):
+        subprocess.run(['python3', '-c', 'import torch'])
```

Comment:
Always use `sys.executable`, since `python3` in the path might be different from Python runtime used to run this test
```suggestion
        subprocess.run([sys.executable, '-c', 'import torch'])
```